ROADMAP.md

```diff
@@ -1,74 +1,121 @@
-# Roadmap:  November 2021 - January 2022
+# Roadmap:  February 2022 - April 2022
 
 Every few months, the team will highlight areas of focus for our work and upcoming research.
 
 We select items for inclusion in the roadmap from the Top Community Issues, [Core Services](docs/contributing/core-services.md), and internal priorities. Where community sourced contributions exist we will work with the authors to review and merge their work. Where this does not exist or the original contributors are not available we will create the resources and implementation ourselves.
 
 Each weekly release will include necessary tasks that lead to the completion of the stated goals as well as community pull requests, enhancements, and features that are not highlighted in the roadmap. To view all the items we've prioritized for this quarter, please see the [Roadmap milestone](https://github.com/hashicorp/terraform-provider-aws/milestone/138).
 
-This roadmap does not describe all the work that will be included within this timeframe, but it does describe our focus. We will include other work as events occur .
+This roadmap does not describe all the work that will be included within this timeframe, but it does describe our focus. We will include other work as events occur.
 
-In the period spanning August to October 2021, 573 Pull Requests were opened in the provider and 465 were merged, adding support for:
+In the period spanning November 2021 to Janury 2022, 627 Pull Requests were opened in the provider and 506 were merged, adding support for:
```

Comment:
@breathingdust these are stats as of today, will need to be updated end of week!